src/config/selectLanguage.js

```diff
@@ -0,0 +1,22 @@
+const readlineSync = require("readline-sync");
```

Comment:
Hey bro. So we are not going to be needing this file. The implementation i will prefer is to have the user decide at the point of running the program. So if you run ```yorl test.yl``` then the default warning language should be english and if you prefer your warnings in another lang, you'll have ```yorl test.yl -w yoruba``` and your default will update accordingly.
So i've pushed a pr for the implementation of ```yorl test.yl -w yoruba``` in cli.js. so you can continue from there

Reply:
Great! I would make another pr soon